neos-spring-java/src/main/java/school/sptech/neosspringjava/api/controllers/ProductController.java

```diff
@@ -70,13 +70,13 @@ public ResponseEntity<ProductResponse> updateProductStatus(@PathVariable Integer
         return ResponseEntity.ok(updatedProduct);
     }
 
-    @GetMapping("/by-establishment/status/name")
+    @GetMapping("/by-establishment/{id}/status/name")
```

Comment:
aqui tbm ta usando requestParam no ID quando deveria ser só o Status, o que for id deve ser PathVariable